src/zino/trapobservers/ignored_traps.py

```diff
@@ -0,0 +1,23 @@
+"""This module implements a trap observer to ignore spammy traps"""
+import logging
+from typing import Optional, Set
+
+from zino.trapd import TrapMessage, TrapObserver, TrapType
+
+_logger = logging.getLogger(__name__)
+
+
+class IgnoreTraps(TrapObserver):
+    """Completely ignores spammy traps observed in Uninett, that we don't care about"""
+
+    WANTED_TRAPS: Set[TrapType] = {
+        ("BGP4-MIB", "bgpBackwardTransition"),
+        ("BGP4-MIB", "bgpBackwardTransNotification"),
+        ("BGP4-V2-MIB-JUNIPER", "jnxBgpM2BackwardTransition"),
+        ("SNMPv2-MIB", "authenticationFailure"),
+        ("CISCOTRAP-MIB", "tcpConnectionClose"),
+        ("BGP4-V2-MIB-JUNIPER", "jnxBgpM2BackwardTransition"),
```

Comment:
Duplicate line

Reply:
Ah, that's funny.  This was just moved from the original implementation, where no-one noticed the duplication :laughing: